components/private_channel/browser/private_channel_service.cc

```diff
@@ -0,0 +1,320 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <utility>
+
+#include "brave/components/private_channel/browser/private_channel_service.h"
+#include "brave/components/private_channel/browser/static_values.h"
+#include "brave/components/private_channel/browser/request_utils.h"
+#include "brave/components/private_channel/client_private_channel.h"
+#include "brave/components/private_channel/utils.h"
+
+#include "base/logging.h"
+#include "extensions/common/url_pattern.h"
+#include "net/base/load_flags.h"
+#include "net/traffic_annotation/network_traffic_annotation.h"
+#include "services/network/public/cpp/resource_request.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/net/system_network_context_manager.h"
+
+using namespace brave_private_channel_request_utils;  // NOLINT
+
+namespace brave_private_channel {
+
+  PrivateChannel::PrivateChannel(std::string referral_code) {
```

Comment:
don't indent inside namespace

Reply:
- [x] run `clang_format` to fix indentation issues